docs/book/src/plugins/available-plugins.md

```diff
@@ -1,5 +1,15 @@
-# Available Plugins
+# Available plugins
 
-This section details how to use the currently available plugins in Kubebuilder. 
+This section describes the plugins supported and shipped in with the Kubebuilder project.
 
-  - [Declarative V1](declarative-v1.md)
\ No newline at end of file
+<aside class="note">
+
+<h1>But you can create your own plugins, see:</h1>
+
+- [Creating your own plugins][create-plugins]. 
+- [Plugins Phase 2][plugins-phase-2] - will allow users to use these external options as helpers to perform the scaffolds with the tool.
+
+</aside>
+
+[create-plugins]: creating-plugins.md
```

Comment:
this line should probably be inside of the `aside`?

Reply:
Nops, the link alias should be at the bottom of the docs pages to make it easier we keep the maintained. 
That is the standard.